QuantConnect.TemplateBrokerage.Tests/BrokerageAlgorithmSettings.cs

```diff
@@ -0,0 +1,405 @@
+                        break;
+                    case SecurityType.Forex:
+                        SymbolsToAdd.Add(ForexSymbol);
+                        break;
+                    case SecurityType.Future:
+                        SymbolsToAdd.Add(CanonicalFutureSymbol);
+                        break;
+                    case SecurityType.Cfd:
+                        SymbolsToAdd.Add(CfdSymbol);
+                        break;
+                    case SecurityType.Crypto:
+                        SymbolsToAdd.Add(CryptoSymbol);
+                        break;
+                    case SecurityType.FutureOption:
+                        SymbolsToAdd.Add(CanonicalFutureOptionSymbol);
+                        break;
+                    case SecurityType.IndexOption:
+                        SymbolsToAdd.Add(CanonicalIndexOptionSymbol);
+                        break;
+                    case SecurityType.CryptoFuture:
+                        SymbolsToAdd.Add(CryptoFutureSymbol);
+                        break;
+                }
+            }
+        }
+    }
+
+    public class BaseMarketSymbols
+    {
+        public virtual Symbol EquitySymbol { get; set; }
+        public virtual Symbol ForexSymbol { get; set; }
+        public virtual Symbol CryptoSymbol { get; set; }
+        public virtual Symbol CfdSymbol { get; set; }
+        public virtual Symbol CryptoFutureSymbol { get; set; }
+        public virtual Symbol CanonicalOptionSymbol { get; set; }
+        public virtual Symbol CanonicalIndexOptionSymbol { get; set; }
+        public virtual Symbol CanonicalFutureSymbol { get; set; }
+        public virtual Symbol CanonicalFutureOptionSymbol { get; }
+    }
+
+    public class DefaultMarketSymbols : BaseMarketSymbols
+    {
+        public override Symbol EquitySymbol { get; set; } = Symbol.Create("AAPL", SecurityType.Equity, Market.USA);
+        public override Symbol ForexSymbol { get; set; } = Symbol.Create("EURUSD", SecurityType.Forex, Market.Oanda);
+        public override Symbol CryptoSymbol { get; set; } = Symbol.Create("BTCUSDT", SecurityType.Crypto, Market.Bybit);
+        public override Symbol CfdSymbol { get; set; } = Symbol.Create("XAUUSD", SecurityType.Cfd, Market.Oanda);
+        public override Symbol CryptoFutureSymbol { get; set; } = Symbol.Create("BTCUSDT", SecurityType.CryptoFuture, Market.Bybit);
+        public override Symbol CanonicalOptionSymbol { get; set; } = Symbol.Create("AAPL", SecurityType.Option, Market.USA);
+        public override Symbol CanonicalIndexOptionSymbol { get; set; } = Symbol.Create("SPX", SecurityType.IndexOption, Market.USA);
+        public override Symbol CanonicalFutureSymbol { get; set; } = Symbol.Create("ES", SecurityType.Future, Market.CME);
+        public override Symbol CanonicalFutureOptionSymbol => Symbol.CreateOption(CanonicalFutureSymbol, Market.CME, OptionStyle.American, OptionRight.Put, 1000, new DateTime(2024, 1, 1));
+    }
```

Comment:
This could be static properties of `BaseMarketSymbols`. `BaseMarketSymbols.DefaultEquitySymbol`, etc.

Reply:
Hmm not sure about this one since we cannot override a static property and for some brokerages we need that the securities that are not supported are null